ssg/build_cpe.py

```diff
@@ -170,13 +170,18 @@ def to_xml_element(self, cpe_oval_filename):
         cpe_item_check.text = self.check_id
         return cpe_item
 
+    @classmethod
+    def from_yaml(cls, yaml_file, env_yaml=None, product_cpes=None):
+        cpe_item = super(CPEItem, cls).from_yaml(yaml_file, env_yaml, product_cpes)
+        if cpe_item.is_product_cpe:
+            cpe_item.is_product_cpe = convert_string_to_bool(cpe_item.is_product_cpe)
```

Comment:
Why are we doing this 'true' -> True conversion? Can't we just use boolean in the YAML file?

Reply:
https://github.com/ComplianceAsCode/content/issues/9749